test/test/facet/test_inspection.py

```diff
@@ -43,12 +45,23 @@ def test_regressor_selector(
         RegressorPipelineDF[LGBMRegressorDF], GridSearchCV
     ]
 ) -> None:
+    print(regressor_selector.summary_report())
```

Comment:
Do we want to keep the `print` statement in this test? Especially that it prints the setup of the passed fixture object, rather than test-specific information.

Reply:
Ah, I see that this fixture is actually used only in this test. Then I can agree to leave it.

Reply:
Ah, I see that this fixture is used only in this test. Then I can agree to leave it.

Reply:
Good call @mtsokol , and the fixture is used more than once. I removed the print statement and instead added a log.debug(…) call inside the fixture.